packages/osd-monaco/src/monaco.d.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+declare module 'monaco-editor/min/vs/editor/editor.main' {
+  export * from 'monaco-editor';
+}
+
+declare module 'monaco-editor/min/vs/base/browser/ui/codicons/codicon/codicon.css' {
+  const content: any;
```

Comment:
are there better types than `any`?

Reply:
The CSS imports are used with a side-effect import syntax (import 'monaco-editor/min/vs/editor/editor.main.css';), which means the content isn't actually assigned to a variable. The type doesn't matter much in this case since it's just for allowing the import to work without TypeScript errors.

Reply:
But I could update it to string.